sunbeam-python/sunbeam/provider/maas/client.py

```diff
@@ -426,3 +426,24 @@ def get_ip_ranges_from_space(client: MaasClient, space: str) -> dict[str, list[d
         if len(ranges) > 0:
             ip_ranges[subnet["cidr"]] = ranges
     return ip_ranges
+
+
+def get_ifnames_from_spaces(client: MaasClient, spaces: set, **extra_args) -> set:
```

Comment:
```suggestion
def get_ifnames_from_spaces(client: MaasClient, spaces: set[str], **extra_args) -> set[str]:
```

Reply:
Return type is modified in subsequent push